submissions/Ant-C-tech/friends-app/js/app.js

```diff
@@ -0,0 +1,288 @@
+'use strict'
+
+const MAIN = document.querySelector('#main')
+const MUSIC_BTN = document.querySelector('#musicBtn')
+const SEARCH = document.querySelector('.search')
+const SEARCH_INPUT = document.querySelector('#icon_prefix')
+const RESET_BTN = document.querySelector('.resetBtn')
+
+const API_LINK = 'https://randomuser.me/api/'
+
+const APP_DELAY = 3000
```

Comment:
No need in keeping this in global scope because you are using it in one function. Just place it there.

Also, this is not "app delay", it is delay for repeating the request ;)